lib/preprocess.js

```diff
@@ -11,10 +11,11 @@ const debug = createDebug('spark:preprocess')
 
 export class Measurement {
   /**
+   * @param {Partial<import('./round.js').RoundData>} r
```

Comment:
Since we're already passing `round.pointerize` as the last argument, I suggest we also pass `round.index` instead of `round`, for consistency and to prevent redundancy